### PR TITLE
Add backend interface for `dbStats`

### LINKED ISSUE #3259: Implement `dbStats` and `collStats` for SQLite
### What should be done?

TBD

### Where?

TBD

### Definition of Done

- all handlers updated;
- unit tests added/updated;
- integration/compatibility tests added/updated;
- spot refactorings done;
- user documentation updated or an issue to create documentation created;
- something else?

### PR DESCRIPTION
**Pull request by chilagrow**
# Description

Closes #3259.

This PR is about backend interface used by `dbStats`. Implementation will be done in another PR.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [x] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/backends/database.go [hunk 1]
```diff
@@ -40,6 +40,8 @@ type Database interface {
 	CreateCollection(context.Context, *CreateCollectionParams) error
 	DropCollection(context.Context, *DropCollectionParams) error
 	RenameCollection(context.Context, *RenameCollectionParams) error
+
+	DBStats(context.Context) (*DBStatsResult, error)
```
Comment (AlekSi) [inline internal/backends/database.go:44]:
```suggestion
	Stats(context.Context, *StatsParams) (*StatsResult, error)
```

It is a Database method, so there is no need for a `DB` prefix. And `StatsParams` should be present, even if they are empty for now.

File: internal/backends/database.go [hunk 2]
```diff
@@ -184,6 +186,33 @@ func (dbc *databaseContract) RenameCollection(ctx context.Context, params *Renam
 	return err
 }
 
+// DBStatsResult represents the results of Database.DBStats method.
+//
+// MongoDB returns "numbers" that could be int32 or int64.
+// FerretDB always returns int64 for simplicity.
```
Comment (AlekSi) [inline internal/backends/database.go:192]: That belongs to the handler, not there

File: internal/backends/database.go (hunk 1 above)
Comment (chilagrow) [reply internal/backends/database.go]: Thanks :+1:

File: internal/backends/database.go (hunk 2 above)
Comment (chilagrow) [reply internal/backends/database.go]: True, removed :+1:

**Review by AlekSi — APPROVED** [review]

**Review by AlekSi — COMMENTED** [review]
It should be `task fmt`-ed

**Review by rumyantseva — DISMISSED** [review]
I see that `fmt` failed, but the new code looks consistent with the rest of backend database code to me.

**Review by rumyantseva — APPROVED** [review]
:shipit:

**Review by noisersup — APPROVED** [review]
LGTM

**Comment by chilagrow** [comment]
@Mergifyio update

**Review by AlekSi — APPROVED** [review]

**Pull request merged**